Gopkg.toml

```diff
@@ -223,3 +223,7 @@ required = [
 [[constraint]]
   name = "go.etcd.io/etcd"
   version = "3.4.9"
+
+[[constraint]]
+  name = "github.com/sirupsen/logrus"
```

Comment:
I chose logrus because it was easier to have pretty, well-indented log output. The installer is something that should, in my opinion, be pleasing to the eyes and nice looking.